src/components/CreateCredential.js

```diff
@@ -1,69 +1,134 @@
+import { React } from 'react';
 import { GetCredential } from '@adobe/gatsby-theme-aio/src/components/GetCredential';
-import { graphql, useStaticQuery } from 'gatsby';
-import React from 'react';
-import "./Credential.css";
-
-const CreateCredential = () => {
-  const data = useStaticQuery(
-    graphql`
-    query { 
-      allFile(filter: {name: {eq: "DemoCode"}}) {
-        edges {
-          node {
-            publicURL
-            name
-          }
-        }
-      }
-    }
-  `
-  );
+import creativeCloud from "../pages/assets/cc-icon.png";
+
+const GetCredentialApiKey = () => {
+
+  const { GATSBY_TEMPLATE_ID, GATSBY_PRODUCT_NAME } = process.env;
```

Comment:
Nice!

Reply:
Do these env vars need to be added to the `deploy.yml`? 

https://github.com/AdobeDocs/cc-everywhere/blob/main/.github/workflows/deploy.yml#L153

Reply:
yes